games-strategy/crimson/additional-files/crimson.rdef

```diff
@@ -0,0 +1,14 @@
+
+resource app_flags B_SINGLE_LAUNCH | B_ARGV_ONLY;
+
+resource app_version {
+	major  = @MAJOR@,
+	middle = @MIDDLE@,
+	minor  = @MINOR@,
+	variety = B_APPV_FINAL,
+	internal = 0,
+	short_info = "CrimsonFields",
+	long_info = "A turn-based tactical war game"
+};
+
+resource app_signature "application/x-SDL-executable";
```

Comment:
Could you change that to a unique, descriptive and conforming signature, like "application/x-vnd.CrimsonFields"?